packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -175,6 +175,11 @@ export interface FieldState {
    * @internal -- we will try to make this unnecessary
    */
   origin?: DataFrameFieldIndex;
+
+  /**
+   * Boolean value is true if field is in a larger data set with multiple frames.
+   */
+  multipleFrames?: boolean;
```

Comment:
This is not strictly true, as this only is set in calculateFieldDisplayName. Should add to the comment that this only related to the cached displayName